.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: ruby
```

Comment:
This doesn't matter since we don't use travis. I want to try merging this fork back into robhurrings though once this PR is closed so we can switch back to his branch.